go.mod

```diff
@@ -7,23 +7,22 @@ require (
 	github.com/Microsoft/go-winio v0.4.15
 	github.com/aws/aws-sdk-go v1.35.24
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/containerd/containerd v1.4.12
 	github.com/containerd/ttrpc v1.0.2
 	github.com/containerd/typeurl v1.0.2
-	github.com/docker/distribution v2.8.0+incompatible // indirect
-	github.com/docker/docker v20.10.12+incompatible
+	github.com/docker/distribution v2.8.1+incompatible // indirect
+	github.com/docker/docker v20.10.17+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/docker/go-units v0.4.0
 	github.com/euank/go-kmsg-parser v2.0.0+incompatible
+	github.com/gogo/googleapis v1.4.1
```

Comment:
Why do we need to take on a new dependency by removing containerd?

Reply:
See above ^ we really need to update docker/docker and docker/distribution, so that drags this in.

Reply:
hmm... do we have a good sense for the stability/maintainedness of this new gogo repo? gogo/protobuf has me wary

Reply:
@liggitt will dig into this for sure

Reply:
`gogo/googleapis` is also being used in the generated [introspection.pb.go](https://github.com/google/cadvisor/blob/master/third_party/containerd/api/services/introspection/v1/introspection.pb.go#L10)

Reply:
@akhilerm exactly!

@liggitt the `main` branch of containerd/containerd no longer has this. It was removed in https://github.com/containerd/containerd/pull/6841

Here's how it looks now:
https://github.com/containerd/containerd/blob/main/api/services/introspection/v1/introspection.pb.go

So when containerd 1.7.x series starts we can switch to it and the dependency will get cleaned up.